### DISCUSSION TITLE
<Editor/> is being re-rendered every time we hover on any of the node on canvas.

### DISCUSSION THREAD

**Discussion by ProPhycient5** [prevwong/craft.js#465] [Show and tell]
I have verified from the source code of Landing example that Whenever I hover on any of the component/node on the canvas, the entire `<Editor/>` gets re-rendered every time. So how can I restrict this performance issue ????

**Comment by Speedwares** [comment]
Hey @ProPhycient5 did you later find a way around this?
I just discovered the same thing recently and it's becoming a performance bottleneck as I am adding more components inside the Editor and resolver.

**Comment by ProPhycient5** [comment]
No, @Speedwares. I did not find any solid reason why this keeps happening although I did a lot of investigation. Yes, that's true, it impact the performance of the webapp.

**Reply by Speedwares** [reply]
Thanks for the response @ProPhycient5.  

Hey @prevwong, would really appreciate your input on this. Thank you for your time!

**Comment by hugominas** [comment]
I think, this most be something on your setup, I just teste on a prod website and click and hover only re-renders each component.
![educacaoambiental cascais pt](https://github.com/prevwong/craft.js/assets/3314986/a5e98dde-31a1-436a-a097-adc84794069a)
